app/constant.js

```diff
@@ -53,10 +53,14 @@ const MESSAGE = {
 
 const URL = {
   https: 'https://',
+  npmPackageUrl: 'www.npmjs.com/package/fetch/v/',
```

Comment:
1. start url with https://
2. why package name is hard coded in constant file ?